src/js/components/generateEF/GenerateEFContent.jsx

```diff
@@ -8,7 +8,7 @@ import React from 'react';
 import GenerateEFOverlay from './GenerateEFOverlay';
 import GenerateEFItem from './generateItem/GenerateEFItem';
 
-export default class GenerateFilesContent extends React.Component {
+export default class GenerateEFContent extends React.Component {
```

Comment:
Im super perplexed how this was working at all...

Reply:
As long as you name it when you import it doesn't really matter. Names are suggestions.